### DISCUSSION TITLE
Access parent scope data in x-if

### DISCUSSION THREAD

**Discussion by ArmeniaH** [alpinejs/alpine#1797] [1. Help] [answered]
```
<template x-for="a in someArr">
  <div>
    <template x-if="a.someProp">
      <span x-text="a.someOtherProp>aaa</span>
    </template>
  </div>
</template>
```

Seems like a.someOtherProp is not working and the value of span is always aaa.
How can I achieve this?

**Comment by danddanddand** [comment, marked as answer]
I'm not sure what you're doing wrong but this works. Added your code to codepen (fixing where you aren't closing x-text attribute) and seems fine. https://codepen.io/danddanddand/pen/MWmpZPj

**Reply by ArmeniaH** [reply]
Thanks, this is weird, I will dig more and update the discussion.

**Comment by ArmeniaH** [comment]
Ok so due to more digging, I figured out that I was using an outdated lib code for alpineJS.

So I just downloaded the contents of alipineJS CDN and saved it in a local JS file, seems like there was an issue with it.

Weird thing was that everything else that I used in the project was working correctly.
